### ISSUE TITLE
FullText print Whosh diagnostics to terminal but doesn't ID Gramplet

### ISSUE THREAD

**Issue by emyoulation** [kkujansuu/gramps#40]
When starting Gramps, the FullText tool addon initializes and sends some extra messages to the terminal.  But the lack of an identifier for the plugin makes it hard to track down the source of the diagnostic. 

```
[districtsupport@fedora-vm ~]$  flatpak run org.gramps_project.Gramps
whoosh loaded
dbid 
dbid 61f41273
/home/districtsupport/.gramps/disable/grampsdb/61f41273/indexdir exists _GrampsContacts
```

**Comment by kkujansuu** [comment]
Yes, that's true. I will remove those messages. Also noticed today the
Fulltext is badly broken: changes in the Person objects are not stored in
the fulltext database at all. Will fix this also.

**Comment by emyoulation** [comment]
Clicked the Close Issue button by accident

**Comment by kkujansuu** [comment]
Both issues fixed now (version 0.9.1).

**Comment by emyoulation** [comment]
Nice!  My Gramps starts in the terminal with zero warnings now!

Thank you!
